COLLABORATOR_GUIDE.md

```diff
@@ -294,22 +288,121 @@ After generating one, make sure to keep it safe because GitHub will not show the
 
 ### Doing a release
 
-To do a release, check out the master branch
+It is also recommended you have a clean clone of Video.js for each release line you want to relase.
+That means having a folder for master/v6 and one for 5.x.
+
+```sh
+# for v6
+git clone git@github.com:videojs/video.js.git videojs-6-release
+# for v5
+git clone git@github.com:videojs/video.js.git videojs-5-release
+```
+
+#### Video.js 6
+
+Make sure go to the master branch and grab the latest updates.
 
 ```sh
 git checkout master
+git pull origin master
 ```
 
-Then run the contrib command to do the next release. Don't forget to provide your GitHub token so the GitHub release goes through.
```

Comment:
It could also be a good idea to specify the removal of node_modules, reinstallation via `npm i` and then running `npm test` just to make sure everything is okay between this step and `npm publish`